content/media-player/modeling/run_dbt_package.md

```diff
@@ -0,0 +1,144 @@
++++
+title= "Set-up and run dbt package"
+weight = 2
+post = ""
++++
+
+> This step assumes you have data in the `ATOMIC.SAMPLE_EVENTS_MEDIA_PLAYER` table which will be used to demonstrate how to set-up and run the snowplow_media_player dbt package to model Snowplow media player data.
+
+***
+
+#### **Step 1:** Override the dispatch order in your project
+
+To take advantage of the optimized upsert that the Snowplow packages offer you need to ensure that certain macros are called from `snowplow_utils` first before `dbt-core`. This can be achieved by adding the following to the top level of your `dbt_project.yml` file:
+
+```yaml
+# dbt_project.yml
+...
+dispatch:
+  - macro_namespace: dbt
+    search_order: ['snowplow_utils', 'dbt']
+```
+
+If you do not do this the package will still work, but the incremental upserts will become more costly over time.
+
+#### **Step 2:** Adding the `selectors.yml` file
+
+The snowplow_media_player package provides a suite of suggested selectors to run and test the models.
+
+These are defined in the [selectors.yml](https://github.com/snowplow/dbt-snowplow-media-player/blob/main/selectors.yml) file within the package, however to use these model selections you will need to copy this file into your own dbt project directory.
+
+This is a top-level file and therefore should sit alongside your `dbt_project.yml` file.
+
+#### **Step 3:** Set-up Variables
+
+The snowplow_media_player dbt package comes with a list of variables specified with a default value that you may need to overwrite in your own dbt project’s `dbt_project.yml` file. For details you can have a look at our [docs](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/media-player/) which contains descriptions and default values of each variable, or you can look in the installed package’s project file which can be found at `[dbt_project_name]/dbt_packages/snowplow_media_player/dbt_project.yml`.
+
+If you are using the provided sample data in `ATOMIC.SAMPLE_EVENTS_MEDIA_PLAYER`, add the following snippet to the `dbt_project.yml`:
+
+```yaml
+vars:
+  snowplow_media_player:
+    snowplow__start_date: '2023-08-04'
+    snowplow__events_table: SAMPLE_EVENTS_MEDIA_PLAYER
+    snowplow__enable_media_ad: true
+    snowplow__enable_media_ad_break: true
+    snowplow__enable_ad_quartile_event: true
+```
+
+If you are using your own events, depending on which media plugin or tracking implementation you use, you will need to enable the relevant contexts in your `dbt_project.yml`:
+
+{{< tabs groupId="tracking_implementation" >}}
+{{% tab name="Media or Vimeo JS plugin" %}}
+
+The package enables the v2 schema by default so no need to change the variables here, unless you want to enable ads or disable the media session context
```

Comment:
On initial read I understood that you can just copy the below code as is. Perhaps it would help if it was rephrased a bit or perhaps just by adding a semi colon at the end so that we know the below code is for the second part of the sentence.

Reply:
On second thoughts I think I'll just remove this sentence entirely!